app/konnect/org-management/audit-logging/webhook.md

```diff
@@ -0,0 +1,173 @@
+
+     For example, if you are setting up the webhook for Splunk, you could provide a Splunk access token: 
+     `"authorization":"Splunk example-token12234352535235"`.
+        
+    * **Log Format**: The output format of each log message. Can be CEF or JSON.
+1. Switch the toggle to `Enabled`, then save your webhook configuration.
+
+{% endnavtab %}
+{% navtab API %}
+
+Now that you have an external endpoint and authorization credentials, you can set up a webhook in {{site.konnect_short_name}}.
+
+Create a webhook by sending a request to the `/audit-log-webhook` endpoint with the connection details for your SIEM vendor:
+
+```sh
+curl -i -X PATCH https://global.api.konghq.com/v2/audit-log-webhook \
+    --header "Content-Type: application/json" \
+    --header "Authorization: Bearer TOKEN" \
+    --data '{"endpoint":"https://example.com/audit-logs","enabled":true,"authorization":"Bearer example-token","log_format":"cef"}'
+```
+
+Replace the following placeholders with your own data:
+* `TOKEN`: A {{site.konnect_short_name}} [personal access token](https://cloud.konghq.com/global/tokens) or 
+  [system account token](/konnect/org-management/system-accounts/).
+* `"endpoint":"https://example.com/audit-logs"`: The external endpoint that will receive audit log messages. 
+   
+   {:.note}
+    > Check your SIEM documentation to find out where to send CEF data.
+    
+* `"authorization":"Bearer example-token"`: The authorization type and credential to pass to your log collection endpoint. 
+{{site.konnect_short_name}} will send this string in the `Authorization` header of requests to that endpoint.
+
+    For example, if you are setting up the webhook for Splunk, you could provide a Splunk access token: `"authorization":"Splunk example-token12234352535235"`.
+
+* `log_format`: The output format of each log message. Can be `cef` or `json`.
+
+If the request is successful, you will receive a `200` response code, and a response body containing the webhook's configuration details: 
+
+```json
+{
+    "endpoint":"https://example.com/audit-logs",
+    "log_format":"cef",
+    "enabled":true,
+    "updated_at":"2023-04-01T00:00:01Z"
+}
+```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Your webhook should now start receiving audit logs. 
+
+## View webhook configuration and status
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+You can view the status of your webhook through the **Audit Logs Setup** page under 
+{% konnect_icon organizations %} **Organization**.
+
+Notice the status badge next to title of the webhook. For example, the following webhook is active:
+
+![Audit log webhook](/assets/images/docs/konnect/konnect-audit-log-webhook.png)
+
+To find the last attempt timestamp and the last response code, use the audit log API.
+
+{% endnavtab %}
+{% navtab API %}
+
+View your audit log webhook configuration by running the following command:
+
+```sh
+curl https://global.api.konghq.com/v2/audit-log-webhook \
```

Comment:
Could we update the endpoint url in this example to be `https://us.api.konghq.com/v2/audit-log-webhook`
or `https://eu.api.konghq.com/v2/audit-log-webhook` since these are region-specific?